tests/data/input/dflowfm_individual_files/crsdef1.ini

```diff
@@ -15,7 +15,7 @@
     type                  = circle
     thalweg               = 0.0
     diameter              = 2.0
-    frictionType          = 3
```

Comment:
Is this actually a IntEnum?

Reply:
No, it's not. In user input this is definitely a string enum. Only later in kernel it becomes an int enum, but that is not relevant for us here.